boilerplate/app/navigators/navigation-utilities.ts

```diff
@@ -104,7 +104,9 @@ export function useNavigationPersistence(storage: any, persistenceKey: string) {
 
     if (previousRouteName !== currentRouteName) {
       // track screens.
-      __DEV__ && console.tron.log(currentRouteName)
+      if (__DEV__) {
+        console.tron.log(currentRouteName)
+      }
```

Comment:
This form of `if (__DEV__) {` is better because Metro will literally strip the whole thing out on compilation (unlike the `__DEV__ && ...` method).